### DISCUSSION TITLE
staticman

### DISCUSSION THREAD

**Discussion by whatisityouseek** [mmistakes/minimal-mistakes#2920] [Q&A]
I'm trying to get the staticman comments setup.  Everything seems to be working, but I'm getting a CORS issue.  I'm running the staticman app on heroku, maybe I need to do something on the heroku side to enable CORS?  Has anyone else had this issue?

**Comment by whatisityouseek** [comment]
Looks like this is just a weird error message i'm getting from the browser.  When I looked at the heroku logs the app wasn't booting properly so that was the issue.  Not actually related to CORS at all

**Reply by dollardhingra** [reply]
I too got this issue when I was trying to install staticman. Using utterances for now.
